main.go

```diff
@@ -104,7 +106,11 @@ func main() {
 		if *shortFlag {
 			flagsCommands = flagsCommands + " -short"
 		}
-		coverageCommand = fmt.Sprintf("%s %s && %s", testCommand, flagsCommands, toolCoverCommand)
+		rootFolder := "."
+		if *rootFolderFlag != "" {
```

Comment:
I'm not certain, but wondering if this conditional might be unnecessary. On line 78 I think the default value for the rootFolderFlag variable is set to ".", so if this if statement is checking for a default empty value like I think it is, it's probably unnecessary. Please correct me if I'm misunderstanding!

Reply:
Good catch! I must have auto-piloted and borrowed the pattern from line 96
```
	if *coverageCommandFlag != "" {
		coverageCommand = *coverageCommandFlag
		if *tagsFlag != "" || *shortFlag {
			log.Println("Warning: When the covercmd flag is used the -tags and -short flags will be ignored.")
		}
	} else { ... truncated
```
I removed it and It still works for me.